### PR TITLE
Add once

### PR DESCRIPTION
**Pull request by balloob**
Add `once` helper to listen to an event exactly once.

### CODE REVIEW

**Comment by marcelveldt** [comment]
Looks fine to me!

File: zwave_js_server/event.py [hunk 1]
```diff
@@ -35,6 +35,18 @@ def unsubscribe() -> None:
 
         return unsubscribe
 
+    def once(self, event_name: str, callback: Callable) -> Callable:
+        """Listen for an event exactly once."""
+        unsub = None
```
Comment (MartinHjelmare) [inline zwave_js_server/event.py:40]: This isn't needed I think.

File: zwave_js_server/event.py [hunk 2]
```diff
@@ -35,6 +35,18 @@ def unsubscribe() -> None:
...
+
+        def event_listener(event):
```
Comment (MartinHjelmare) [inline zwave_js_server/event.py:42]:
```suggestion
        def event_listener(event: str) -> None:
```

File: zwave_js_server/event.py [hunk 3]
```diff
@@ -1,7 +1,7 @@
 """Provide Event base classes for Z-Wave JS."""
 import logging
 from dataclasses import dataclass, field
-from typing import Callable, Dict, List
+from typing import Any, Callable, Dict, List
```
Comment (MartinHjelmare) [inline zwave_js_server/event.py:4]:
```suggestion
from typing import Callable, Dict, List
```

**Review by MartinHjelmare — APPROVED** [review]

**Pull request merged**